openid/jwttokenvalidatormock_test.go

```diff
@@ -24,7 +25,7 @@ type validateResp struct {
 	err error
 }
 
-func (j *jwtTokenValidatorMock) validate(t string) (*jwt.Token, error) {
+func (j *jwtTokenValidatorMock) validate(r *http.Request, t string) (*jwt.Token, error) {
 	j.Calls <- &validateCall{t}
```

Comment:
Also include the http Request in the validateCall and update one of the tests to assert the correct value of the request (see previous comments)